libs/core/src/components/sage-button/stories/sage-button.stories.mdx

```diff
@@ -0,0 +1,134 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+<Meta title="webcomponents/sage-button" component="sage-button" argTypes={extractArgTypes('sage-button')} />
+
+export const Template = (args) =>
+  html` <sage-button value=${args.value} variant=${args.variant} name=${args.name} icon=${args.icon} disabled=${args.disabled} type=${args.type}> ${args.slot} </sage-button> `;
+
+# sage-button
+
+A button component to generate all versions of `sage-button` as outlined in [Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?node-id=3%3A15&t=dIcvX1VtOu0FL0VE-0).
+
+## Usage Guidelines
```

Comment:
The only thing I can think of to add here could be some guidelines and best practices around accessibility, but again, we can always revisit docs when we ave our retro.

Reply:
Added a section, feel free to note anything you think could be included other than what I've added!